### DISCUSSION TITLE
JBOSS Support In Skywalking Java Agent

### DISCUSSION THREAD

**Discussion by AjinThankachan** [apache/skywalking#9263] [Q&A]
Hi Team,
Skywalking Version :8.9.1
Java Agent Version  : 8.8.0
Will Skywalking support the JBOSS java applications. I am doing one test in one EAP 7 JBOSS Application. It is throwing error in application level when adding the java agent in JAVA_OPTS. Any one has done such testing with similar applications. If so can someone help me.

Thanks

**Comment by wu-sheng** [comment]
I think all tested supported frameworks and middlewares are listed in the doc.
